### DISCUSSION TITLE
Remove top of the page AI banner

### DISCUSSION THREAD

**Discussion by mirkodrummer** [mdn/mdn-community#474] [MDN Plus] [answered]
First of all, I'm more than happy MDN Plus subscriber and a few other Mozilla products. So, many thanks!
A little feedback: is it possible to let the user dismiss, or remove altogether, the "Get real-time assistance with your coding queries. Try AI Help now!" banner at the top of the mdn site? It's a bit annoying and takes vertical real estate.

**Comment by HerminaC** [comment]
Hi @mirkodrummer ! First of all, thank you for being an MDN Plus subscriber.
Subscribers will be able to dismiss any banner on MDN, in the same fashion you can now opt out of ads. We cannot land new code before January 3rd, but this is something we'll prioritize in January 2024.

Regards!

**Reply by mirkodrummer** [reply]
@HerminaC Hi, seems like subscribers can't dismiss any banner on MDN yet. Any update on this?

**Comment by HerminaC** [comment, marked as answer]
Thanks for the reminder, @mirkodrummer  and apologies for letting this slide. We’ve addressed this issue by integrating it with our ad-free functionality. Starting tomorrow, if you’re already ad-free, you won’t see the top banner anymore. If you're still seeing ads, check your account settings and opt for the ad-free experience option to remove them. 

Best,
Hermina
